src/material/datepicker/datepicker-base.ts

```diff
@@ -232,10 +232,19 @@ export interface MatDatepickerControl<D> {
   stateChanges: Observable<void>;
 }
 
+/** A picker panel that can be associated with a {@link MatDatepickerControl}. */
+export interface MatDatepickerPanel<C extends MatDatepickerControl<D>, S,
```

Comment:
Wouldn't something else in the library need to change from hard-coding e.g. `MatDatepickerBase` to `MatDatepickerPanel`?

Reply:
Yes, I neglected to upload key changes. It's fixed now.